apps/judicial-system/backend/src/app/modules/case/models/case.model.ts

```diff
@@ -958,4 +958,14 @@ export class Case extends Model {
   @HasMany(() => IndictmentCount, 'caseId')
   @ApiPropertyOptional({ type: IndictmentCount, isArray: true })
   indictmentCounts?: IndictmentCount[]
+
+  /**********
+   * Indicates whether the prosecutor requests a drivers license suspension
+   **********/
+  @Column({
+    type: DataType.BOOLEAN,
+    allowNull: false,
```

Comment:
Why don't we want to allow null for this column?

Reply:
😱 we do want nulls. Will fix.